src/tokens.ts

```diff
@@ -534,6 +534,8 @@ export namespace Git {
     export interface IContext {
       currentRef: string | SpecialRef;
       previousRef: string | SpecialRef;
+      // Used only during merge conflict diffs
+      baseRef?: string | SpecialRef;
```

Comment:
```suggestion
      baseRef?: string;
```

The base reference cannot be the index or the head.